plugins/modules/nxos_vxlan_vtep_vni.py

```diff
@@ -84,21 +84,30 @@
     choices:
     - present
     - absent
+    type: list
```

Comment:
seems like the type of `state` key was changed to list?
```suggestion
    type: str
```